frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -86,7 +87,13 @@ export default function WorkflowListView() {
 
   usePolling(loadData, VX_POLLING_INTERVAL);
 
-  const renderRuns = useMemo(
+  const getUserDisplayName = (run: VoxelyticsWorkflowListingRun) => {
+    return run.userFirstName != null || run.userLastName != null
+      ? [run.userFirstName, run.userLastName].join(" ")
+      : run.hostusername;
```

Comment:
I don't see the property `hostusername` on `VoxelyticsWorkflowListingRun`?

Reply:
but it should be there :sweat_smile: as of now, in . 971 in api_flow_types.

Reply:
ok, apparently, I was blind :see_no_evil: by the way, why is it not camel case and `hostUserName`? (similar to `userFirstLastName`)

Reply:
because hostname wasnt camel cased :sweat_smile: I will change it :)